app/src/main/java/org/simple/clinic/drugs/selection/EditMedicinesScreen.kt

```diff
@@ -29,8 +29,11 @@ import org.simple.clinic.drugs.EditMedicinesInit
 import org.simple.clinic.drugs.EditMedicinesModel
```

Comment:
Let's make this change at the end, once all the refill medicine logic is completed.